docs/GettingStarted.asciidoc

```diff
@@ -194,6 +194,24 @@ tags.
 }
 -------------------------------------------------------------------
 
+==== Areas ====
+There are three kinds of areas:
+
+* *Regular areas* define relevant parts of the screenshot. Those must match
+  with at least the specified similarity percentage. Regular areas are
+  displayed as green boxes in the needle editor and as green or red frames
+  in the needle view.
```

Comment:
"and as green or red frames in the needle view. *green* for matching areas, *red* for non-matching ones."